### ISSUE TITLE
anmn_temp_gridded_product.py - Max depth? Time stamp?

### ISSUE THREAD

**Issue by ggalibert** [aodn/data-services#695]
A couple of questions on the anmn_temp_gridded_product scripts. The answers would be worth being included in the NetCDF file:
1. How is the maximum depth over which the vertical interpolation defined? Unfortunately it looks like this is not consistent from one deployment to another.
2. Where does the time stamp lies in the averaging cell? Start/middle/end of the cell? Ideally, it should be middle. Ideally as well, we would like to choose the cell bounds that for a 30min averaging we output 00:00 (bounds 23:45 - 00:15) rather than 00:15 (bounds 00:00 - 00:30).

**Comment by lbesnard** [comment]
The maximum depth can be different for every product generated. See the following function :
https://github.com/aodn/data-services/blob/master/ANMN/temperature_grid_product/anmn_temp_gridded_product.py#L286
The last NON NaN value is kept. This made sense to do so since it was a product. 

The Time bin is defined in this line
https://github.com/aodn/data-services/blob/master/ANMN/temperature_grid_product/anmn_temp_gridded_product.py#L178
In short, it is the middle of the cell.

**Comment by ggalibert** [comment]
Thanks, to complete this post I would add that the averaging cell starts at the first data value so that the time stamps can fall at any time and not necessarily at a round 00 or 15 minute.
